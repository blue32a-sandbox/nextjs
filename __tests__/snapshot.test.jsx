import { render } from '@testing-library/react'
import Snapshot from '../pages/snapshot'
import '@testing-library/jest-dom'

describe('Snapshot', () => {
  it('renders page unchanged', () => {
    const { container } = render(<Snapshot />)
    expect(container).toMatchSnapshot()
  })
})
