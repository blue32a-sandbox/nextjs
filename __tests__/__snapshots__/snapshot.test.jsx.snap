// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders page unchanged 1`] = `
<div>
  <h1>
    This is Snapshot page
  </h1>
  <p>
    This page is snapshot by test.
  </p>
</div>
`;
